onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -181,6 +188,22 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
             return
 
         root_input = other_inputs[0]
+        """
+        Match flaubert                     Mask
+                                            |
+        Mul --> LayerNormalization -->  Attention --> MatMul --> Add
+         |                                                        |
+         |                                                        |
+         +---------------------------------------------------------
+        """
+        mul_before_layernorm = self.model.match_parent(normalize_node, 'Mul', 0)
+        if mul_before_layernorm is not None:
+            mul_children = input_name_to_nodes[mul_before_layernorm.output[0]]
+            if mul_children is not None and len(mul_children) == 2:
```

Comment:
what if this is not matched (like it has more than 2 children, shall we return)?